server/surveyRdb/schemaRdb/dataColProps.js

```diff
@@ -54,6 +60,22 @@ const props = {
     [colTypeProcessor]: () => () => sqlTypes.decimal,
   },
 
+  [nodeDefType.date]: {
+    [colTypeProcessor]: () => () => sqlTypes.date,
+    [colValueProcessor]: (surveyInfo, nodeDefCol, nodeCol) => {
+      const [day, month, year] = Node.getNodeValue(nodeCol, '').split('/').map(trim)
+      return () => isValid(year, month, day) ? `${year}-${month}-${day}` : null
+    }
+  },
+
+  [nodeDefType.time]: {
+    [colTypeProcessor]: () => () => sqlTypes.time,
+    [colValueProcessor]: (surveyInfo, nodeDefCol, nodeCol) => {
+      const [hour, minute] = Node.getNodeValue(nodeCol, '').split(':').map(trim)
+      return () => hour !== '' && minute !== '' ? `${hour}:${minute}:00` : null
```

Comment:
you can use stringUtils.isNotBlank and avoid the .map(trim)

Reply:
Ok